doc/source/release-notes/Percona-XtraDB-Cluster-5.6.27-25.13.rst

```diff
@@ -0,0 +1,78 @@
+.. rn:: 5.6.27-25.13
+
+===================================
+Percona XtraDB Cluster 5.6.27-25.13 
+===================================
+
+Percona is glad to announce the release of |Percona XtraDB Cluster| 5.6.27-25.13 on January 11, 2015. Binaries are available from the `downloads section <http://www.percona.com/downloads/Percona-XtraDB-Cluster-56/release-5.6.27-25.13/>`_ or from our :ref:`software repositories <installation>`.
+
+Based on `Percona Server 5.6.27-75.0 <http://www.percona.com/doc/percona-server/5.6/release-notes/Percona-Server-5.6.27-75.0.html>`_, `Galera Replicator 3.13 <https://github.com/codership/galera/issues?q=milestone%3A25.3.13>`_ and on `Codership wsrep API 25.12 <https://github.com/codership/mysql-wsrep/issues?q=milestone%3A5.6.27-25.12>`_, Percona XtraDB Cluster 5.6.27-25.13 is now the current release. All Percona software is open-source and free. Details of this release can be found in the `5.6.27-25.13 milestone on Launchpad <https://launchpad.net/percona-xtradb-cluster/+milestone/5.6.27-25.13>`_.
+
+For more information about relevant Codership releases, see `this announcement <http://galeracluster.com/2015/11/announcing-galera-cluster-5-5-42-and-5-6-25-with-galera-3-12-2/>`_.
+
+Bugs Fixed
+==========
+
+* `1261688 <https://bugs.launchpad.net/percona-xtradb-cluster/+bug/1261688>`_: Fixed race condition when two skipped replication transactions were rolled back, which caused ``[ERROR] WSREP: FSM: no such a transition ROLLED_BACK -> ROLLED_BACK`` with ``LOAD DATA INFILE``
+
+* `1330941 <https://bugs.launchpad.net/percona-xtradb-cluster/+bug/1330941>`_: Conflict between ``wsrep_OSU_method=RSU`` and ``wsrep_desync=on`` was not considered a bug.
+
+* `1362830 <https://bugs.launchpad.net/percona-xtradb-cluster/+bug/1362830>`_: Corrected ``xtrabackup-v2`` script to consider only the last specified ``log_bin`` directive in :file:`my.cnf`. Multiple ``log_bin`` directives caused SST to fail.
+
+* `1370532 <https://bugs.launchpad.net/percona-xtradb-cluster/+bug/1370532>`_: Toggling ``wsrep_desync`` while node is paused is now blocked.
```

Comment:
use :variable:`$variable_name` instead of `` for variable names as this will automatically create a link to the variable description.